File: New Restaurantelar API/src/main/java/com/salesianos/triana/proyecto/Restaurantelar/dto/CreatePlatoDto.java
package com.salesianos.triana.proyecto.Restaurantelar.dto;


import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CreatePlatoDto {

    private String name;
    private String description;
    private boolean distinct;
    private String imageUrl;
    private double price;



}

File: new_restaurantelar_admin/src/app/models/create_plato_dto.ts
export class CreatePlatoDto {
    name: string;
    description: string;
    distinct: boolean;
    imageUrl: string;
    price: number;
    
    constructor() {
      this.name = '';
      this.description = '';
      this.distinct = false;
      this.imageUrl = '';
      this.price = 0;
    }
  }